tests/unit/test_template.py

```diff
@@ -0,0 +1,118 @@
+import tmt
+import tmt.log
+import tmt.plugins
+import tmt.steps.discover
+import tmt.template
+import tmt.utils
+from tmt.utils import Path
+
+
+def test_init_custom_templates_folder(tmppath: Path) -> None:
+    templates_dir = tmppath / 'templates'
+    template = tmt.template.TemplateManager(templates_dir)
```

Comment:
Let's call it `manager`? Or `template_manager`, it's a bit confusing as it's not a single template.

Reply:
Renamed in eca25386d95a1e4b9f879f3df89d871720787173.